docker-for-mac/troubleshoot.md

```diff
@@ -25,21 +25,18 @@ GitHub](https://github.com/docker/for-mac/issues), or the [Docker for Mac
 forum](https://forums.docker.com/c/docker-for-mac), we can help you troubleshoot
 the log data.
 
-Choose ![whale menu](images/whale-x.png){: .inline} -->
+Choose ![whale menu](images/whale-x.png){: .inline} →
 **Diagnose & Feedback** from the menu bar.
 
 ![Diagnose & Feedback](images/diagnose-feedback.png){:width="600px"}
 
-Select **Diagnose**.  It runs diagnostics, shows results, and uploads the
-results to Docker.  A diagnostic ID is generated, which must be provided when
-communicating with the Docker Team.  Optionally, you can open an issue on GitHub
-using the uploaded results and ID as a basis.
+Once the diagnostics are available, you can upload them and obtain a **Diagnostic ID**, which must be provided when communicating with the Docker team. For more information on our policy regarding personal data you can read [how is personal data handled in Docker Desktop](https://docs.docker.com/docker-for-mac/faqs/#how-is-personal-data-handled-in-docker-desktop).
 
 ![Diagnostics & Feedback with ID](images/diagnose-feedback-id.png){:width="600px"}
 
 If you click **Report an issue**, this opens [Docker for Mac issues on
 GitHub](https://github.com/docker/for-mac/issues/) in your web browser in a
-“create new issue” template, to be completed before submission.
```

Comment:
We can add here : "do not forget to copy/paste your diagnostic ID".

Reply:
@ahh-docker Fixed! ✨